updating/understanding-the-update-service.adoc

```diff
@@ -5,6 +5,8 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, for clusters in a restricted network, there’s no way to access these public APIs to obtain this information. In order to provide a similar user upgrade experience in a restricted network users can install the OpenShift Update Service locally, i.e. accessible within their disconnected network, as described in xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]
```

Comment:
I think that this paragraph is good in the other file (specific to installing/configuring), but that it's not quite appropriate as is here. The focus in this section isn't on the restricted network part. So I'd suggest to stop after the first sentence, and then add a [NOTE] with a more concise version saying that if you're looking to install OSUS locally for a restricted network environment, then see xref:....

It's also missing a period at the end.

Reply:
I took a shot at rewording this, wrapped in a note.